package.json

```diff
@@ -21,6 +21,7 @@
     "affected:lint": "nx affected:lint --parallel",
     "affected:dep-graph": "nx affected:dep-graph",
     "affected": "nx affected",
+    "contributors:add": "all-contributors add",
```

Comment:
How does this work?

Reply:
The script shows a prompt asking info about the contributor and fills the `README.md` and `.all-contributorsrc` files with the needed info.

It  also creates a commit for the changes.

Reply:
So it's just `yarn contributors:add`? No contributor details passed as parameters?

Reply:
No need, the contributor's handle will be prompted. I assumed that it is possible to pass it inline too, but `yarn contributors:add` is enough.